src/views/Scenario/Scenario.js

```diff
@@ -228,7 +243,29 @@ const Scenario = () => {
     label: scenarioListLabel,
     validationStatus: scenarioValidationStatusLabels,
   };
+  // FOR DEVELOPMENT PURPOSE ONLY, DELETE BEFORE MERGE
+  const lastRun = '2023-03-13T16:25:30.795486885Z';
+  // FOR DEVELOPMENT PURPOSE ONLY, DELETE BEFORE MERGE
 
+  const hasScenarioBeenRun = useMemo(
+    () => currentScenario?.data?.lastRun !== null && currentScenario?.data?.state === 'Successful',
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [currentScenario?.data?.lastRun, currentScenario?.data?.state]
+  );
+  const isDashboardSync = useMemo(
+    () => new Date(currentScenario?.data?.lastUpdate) < new Date(lastRun),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [currentScenario?.data?.lastUpdate, lastRun]
+  );
+  const dashboardCardStyle = hasScenarioBeenRun
+    ? {
+        px: 3,
+        '&:last-child': { pb: 2 },
+        bgcolor: () => {
+          if (isDirty || !isDashboardSync) return 'dashboard.warning';
+        },
```

Comment:
Is it ok to have a function for `bgcolor`? In your previous version, it wasn't a function but a string: `{ px: 3, bgcolor: 'dashboard.warning', '&:last-child': { pb: 2 } }`

Reply:
I think it's ok because the function returns a string, can it be a problem ?

Reply:
I just checked in mui docs, and it seems it can work indeed :+1: 
Yet, we could probably make the code simpler by replacing the function by:
`bgcolor: (isDirty || !isDashboardSync) ? 'dashboard.warning' : undefined;`